packages/compose/src/hooks/use-keyboard-shortcut/index.js

```diff
@@ -34,9 +34,13 @@ function isAppleOS( _window = window ) {
 function useKeyboardShortcut( shortcuts, callback, {
 	bindGlobal = false,
 	eventName = 'keydown',
+	isDisabled = false, // This is important for performance considerations.
```

Comment:
It's important, but what is it meant to represent? I was confused when I first saw this, 'til I could understand the context for how it's used in this pull request. It would help to have some documentation for this hook.